qcodes/instrument/channel.py

```diff
@@ -59,10 +60,24 @@ def ask(self, cmd):
     def ask_raw(self, cmd):
         return self._parent.ask_raw(cmd)
 
+    @property
+    def parent(self) -> InstrumentBase:
+        return self._parent
+
     @property
     def root_instrument(self) -> InstrumentBase:
         return self._parent.root_instrument
 
+    @property
```

Comment:
Could we not just simply write
```
return self._parent.name_parts().append(short_name)
```
?

Reply:
Good point. This was written inside out not outside in so it wasn't originally possible